pkg/controller/controlplane/valuesprovider.go

```diff
@@ -188,6 +221,25 @@ var (
 					{Type: &rbacv1.ClusterRoleBinding{}, Name: "extensions.gardener.cloud:provider-aws:aws-custom-route-controller"},
 				},
 			},
+			{
+				Name: aws.AWSLoadBalancerControllerName,
+				Path: filepath.Join(aws.InternalChartsPath, "aws-load-balancer-controller"),
+				Objects: []*chart.Object{
+					{Type: &rbacv1.Role{}, Name: aws.AWSLoadBalancerControllerName + "-leader-election-role"},
+					{Type: &rbacv1.RoleBinding{}, Name: aws.AWSLoadBalancerControllerName + "-leader-election-rolebinding"},
+					{Type: &rbacv1.ClusterRole{}, Name: aws.AWSLoadBalancerControllerName + "-role"},
+					{Type: &rbacv1.ClusterRoleBinding{}, Name: aws.AWSLoadBalancerControllerName + "-rolebinding"},
+					{Type: &corev1.ServiceAccount{}, Name: aws.AWSLoadBalancerControllerName},
+					{Type: &admissionregistrationv1.MutatingWebhookConfiguration{}, Name: aws.AWSLoadBalancerControllerName + "-webhook"},
+					{Type: &admissionregistrationv1.ValidatingWebhookConfiguration{}, Name: aws.AWSLoadBalancerControllerName + "-webhook"},
+					{Type: &v1.PodDisruptionBudget{}, Name: aws.AWSLoadBalancerControllerName},
+					{Type: makeUnstructured(schema.GroupVersionKind{
+						Group:   "elbv2.k8s.aws",
+						Version: "v1beta1",
+						Kind:    "IngressClassParams"}), Name: "alb"},
+					{Type: &networkingv1.IngressClass{}, Name: ""},
```

Comment:
I think this is not needed anymore after the last commit since we don't deploy the params ourselves.

Reply:
dropped